bin/plugin/commands/benchmark.js

```diff
@@ -0,0 +1,278 @@
+/**
+ * External dependencies
+ */
+const fs = require( 'fs' );
+const readline = require( 'readline' );
+const autocannon = require( 'autocannon' );
+const { round } = require( 'lodash' );
+const { table } = require( 'table' );
+const { stringify: csv } = require( 'csv-stringify/sync' );
+
+/**
+ * Internal dependencies
+ */
+const { log, formats } = require( '../lib/logger' );
+
+const OUTPUT_FORMAT_CSV = 'csv';
+
+/**
+ * @typedef BenchmarkOptions
+ *
+ * @property {string} url         An URL.
+ * @property {number} connections A number of multiple requests to make at a time.
+ * @property {number} amount      A number of requests to perform.
+ */
+
+/**
+ * @typedef BenchmarkResults
+ *
+ * @property {Array.<number>} responseTimes An array of response times.
+ * @property {Object}         metrics       The Server-Timing metrics object.
+ */
+
+/**
+ * @typedef BenchmarkCommandOptions
+ *
+ * @property {string} url         An URL to benchmark.
+ * @property {number} concurrency A number of multiple requests to make at a time.
+ * @property {number} number      A number of requests to perform.
+ * @property {string} file        A path to a file with URLs to benchmark.
+ * @property {string} output      An output format.
+ */
+exports.options = [
+	{
+		argname: '-u, --url <url>',
+		description: 'An URL to run benchmark tests for',
+	},
+	{
+		argname: '-c, --concurrency <concurrency>',
+		description: 'Number of multiple requests to make at a time',
+		defaults: 1,
+	},
+	{
+		argname: '-n, --number <number>',
+		description: 'Number of requests to perform',
+		defaults: 1,
+	},
+	{
+		argname: '-f, --file <file>',
+		description: 'File with URLs to run benchmark tests for',
+	},
+	{
+		argname: '-o, --output <output>',
+		description: 'Output format: csv or table',
+		defaults: 'table',
+	},
+];
+
+/**
+ * Runs http benchmarks for an URL.
+ *
+ * @param {BenchmarkCommandOptions} opt Command options.
+ */
+exports.handler = async ( opt ) => {
+	const { concurrency: connections, number: amount } = opt;
+	const results = [];
+
+	for await ( const url of getURLs( opt ) ) {
+		const { completeRequests, responseTimes, metrics } = await benchmarkURL(
+			{
+				url,
+				connections,
+				amount,
+			}
+		);
+
+		results.push( [ url, completeRequests, responseTimes, metrics ] );
+	}
+
+	if ( results.length === 0 ) {
+		log(
+			formats.error(
+				'No URLs were found. Please, provide URLs to benchmark.'
```

Comment:
It would be helpful to include the concrete parameters here which makes this command very intuitive to use (trial and error, you wouldn't even need to read the code or look at help).
```suggestion
				'You need to provide a URL to benchmark via the --url (-u) argument, or a file with multiple URLs via the --file (-f) argument.'
```